discovery-aws-api/src/main/scala/akka/discovery/awsapi/ec2/Ec2TagBasedSimpleServiceDiscovery.scala

```diff
@@ -68,6 +70,33 @@ class Ec2TagBasedSimpleServiceDiscovery(system: ActorSystem) extends SimpleServi
 
   private val runningInstancesFilter = new Filter("instance-state-name", List("running").asJava)
 
+  lazy val defaultClientConfiguration = {
```

Comment:
Why not this:
```suggestion
  private val defaultClientConfiguration = {
```
(or inline it)?